plugins/inputs/opcua/README.md

```diff
@@ -0,0 +1,76 @@
+# Telegraf Input Plugin: opcua_client
+
+The opcua_client plugin retrieves data from OPCUA slave devices
+
+### Configuration:
+
+```toml
+
+# ## Connection Configuration
+#  ##
+#  ## The plugin supports connections to PLCs via OPCUA
+#  ##
+#  ## Device name
+name = "opcua_rocks"
+#
+#  # OPC UA Endpoint URL
+endpoint = "opc.tcp://opcua.rocks:4840"
+#
+#  ## Read Timeout
+#  ## add an arbitrary timeout (seconds) to demonstrate how to stop a subscription
+#  ## with a context.
+timeout = 30
+#
+#  # Time Inteval, default = 10s
+time_interval = "5s"
```

Comment:
The code in opcua_client.go says time_interval defaults to opcua.DefaultSubscriptionInterval.  The gopcua docs say that's 100ms.  https://godoc.org/github.com/gopcua/opcua#pkg-constants

Which default is right? 10s, 5s, 100ms?